jobs/service-fabrik-deployment-hooks/templates/bin/service-fabrik-deployment-hooks_ctl.erb

```diff
@@ -6,6 +6,22 @@ source /var/vcap/packages/bosh-helpers/ctl_setup.sh 'service-fabrik-deployment-h
 
 export PID_FILE=${RUN_DIR}/service-fabrik-deployment-hooks.pid
 export PACKAGE_DIR=${HOME}/packages/service-fabrik-deployment-hooks
+export SECCOMP_CMD=${PACKAGE_DIR}/deployment_hooks/bin/seccomp/seccomp
+
+# installing libseccomp here because .so files are needed at runtime for seccomp executable 
+# installing libseccomp
+pushd ${HOME}/packages/libseccomp
+./configure
+make
+make install
+# set linkings
+ldconfig -v
+popd
+
+# compile add_seccomp_filters script
+pushd ${HOME}/packages/service-fabrik-deployment-hooksdeployment_hooks/bin/seccomp
```

Comment:
is a "/" missing here ?

should it be `pushd ${HOME}/packages/service-fabrik-deployment-hooks/deployment_hooks/bin/seccomp` ?

Reply:
My Bad! Done!